### ISSUE TITLE
The pop-up displaying network requests does not appear in Release 2.0.0 when a modal view or controller is already presented.

### ISSUE THREAD

**Issue by Silviuvr** [pmusolino/Wormholy#145]
The network request pop-up does not appear when a modal view, such as a login bottom sheet, is presented. For example, if I present the login bottom sheet and shake the phone, the pop-up with active requests is not displayed. 

However, after completing the login process and dismissing the bottom sheet, the pop-up works as expected on the main dashboard page.

This behavior suggests the pop-up is not functioning correctly when a modal view is active.

**Comment by robnadin** [comment]
This appears to be a regression. The old version had code to determine the current (top-most) view controller, but in 2.0.0 and later it is simply just trying to present on the root view of the window, regardless of whether that view is already presenting a sheet/modal view itself.

**Comment by akovalov** [comment]
Indeed, 2.0.0 allows presenting the view controller only via a notification. Internal function `presentWormholyFlow()` presents on `UIApplication.shared.windows.first?.rootViewController`.

**Comment by pmusolino** [comment]
This has been fixed here 71b505329781e128265aa4cb3dd949159897fc56, and will be available in the next release

**Comment by pmusolino** [comment]
https://github.com/pmusolino/Wormholy/releases/tag/2.4.0
